docs/topics/proc_exposing-application-metrics-using-prometheus-with-vertx.adoc

```diff
@@ -70,23 +71,22 @@ $ ./prometheus
 <dependency>
   <groupId>io.vertx</groupId>
   <artifactId>vertx-micrometer-metrics</artifactId>
-  <version>3.5.4</version>
 </dependency>
 <dependency>
   <groupId>io.micrometer</groupId>
   <artifactId>micrometer-registry-prometheus</artifactId>
-  <version>1.0.0</version>
```

Comment:
You should have kept this version and only remove `vertx` versions. `vertx-dependencies` provides version alignments for vertx artifacts only.